### ISSUE TITLE
Error when PHP generated code test run against Ruby server

### ISSUE THREAD

**Issue by stanley-cheung** [grpc/grpc#1582]
Ruby server side log
```
stanleycheung@gummybear:~/grpc/src/ruby$ ruby bin/math_server.rb 
 INFO  Object : ... running insecurely on localhost:7071
 INFO  GRPC::RpcServer : handling /math.Math/Div with #<Method: Calculator#div>
 INFO  GRPC::RpcServer : handling /math.Math/DivMany with #<Method: Calculator#div_many>
 INFO  GRPC::RpcServer : handling /math.Math/Fib with #<Method: Calculator#fib>
 INFO  GRPC::RpcServer : handling /math.Math/Sum with #<Method: Calculator#sum>
 INFO  GRPC::RpcServer : waiting: 0, max: 20
 INFO  GRPC::RpcServer : deadline is 1969-12-31 15:59:59 -0800; (now=2015-05-13 14:28:55 -0700)
 INFO  GRPC::Pool : schedule another job
 INFO  GRPC::RpcServer : waiting: 0, max: 20
 INFO  GRPC::RpcServer : deadline is 1969-12-31 15:59:59 -0800; (now=2015-05-13 14:28:56 -0700)
 INFO  GRPC::Pool : schedule another job
 INFO  GRPC::RpcServer : waiting: 0, max: 20
 INFO  GRPC::RpcServer : deadline is 1969-12-31 15:59:59 -0800; (now=2015-05-13 14:28:56 -0700)
 INFO  GRPC::Pool : schedule another job
 INFO  GRPC::RpcServer : waiting: 0, max: 20
 INFO  GRPC::RpcServer : deadline is 1969-12-31 15:59:59 -0800; (now=2015-05-13 14:28:57 -0700)
 INFO  GRPC::Pool : schedule another job
 INFO  Calculator : read <Math::DivArgs: dividend: 1, divisor: 2>
 INFO  GRPC::RpcServer : waiting: 0, max: 20
 INFO  GRPC::RpcServer : deadline is 1969-12-31 15:59:59 -0800; (now=2015-05-13 14:29:08 -0700)
 INFO  GRPC::Pool : schedule another job
 INFO  GRPC::RpcServer : waiting: 0, max: 20
 INFO  GRPC::RpcServer : deadline is 1969-12-31 15:59:59 -0800; (now=2015-05-13 14:29:08 -0700)
 INFO  GRPC::Pool : schedule another job
 INFO  GRPC::RpcServer : waiting: 0, max: 20
 INFO  GRPC::RpcServer : deadline is 1969-12-31 15:59:59 -0800; (now=2015-05-13 14:29:09 -0700)
 INFO  GRPC::Pool : schedule another job
 INFO  GRPC::RpcServer : waiting: 0, max: 20
 INFO  GRPC::RpcServer : deadline is 1969-12-31 15:59:59 -0800; (now=2015-05-13 14:29:09 -0700)
 INFO  GRPC::Pool : schedule another job
 INFO  Calculator : read <Math::DivArgs: dividend: 1, divisor: 2>
```

PHP client side log
```
stanleycheung@gummybear:~/grpc/src/php$ ./bin/run_gen_code_test.sh 
PHPUnit 3.7.28 by Sebastian Bergmann.


Starting test 'GeneratedCodeTest::testSimpleRequest'.
.
Starting test 'GeneratedCodeTest::testServerStreaming'.
.
Starting test 'GeneratedCodeTest::testClientStreaming'.
.
Starting test 'GeneratedCodeTest::testBidiStreaming'.
PHP Fatal error:  Call to a member function getQuotient() on a non-object in /usr/local/google/home/stanleycheung/grpc/src/php/tests/generated_code/AbstractGeneratedCodeTest.php on line 90
PHP Stack trace:
PHP   1. {main}() /usr/local/bin/phpunit:0
PHP   2. PHPUnit_TextUI_Command::main() /usr/local/bin/phpunit:46
PHP   3. PHPUnit_TextUI_Command->run() /usr/share/php/PHPUnit/TextUI/Command.php:129
PHP   4. PHPUnit_TextUI_TestRunner->doRun() /usr/share/php/PHPUnit/TextUI/Command.php:176
PHP   5. PHPUnit_Framework_TestSuite->run() /usr/share/php/PHPUnit/TextUI/TestRunner.php:349
PHP   6. PHPUnit_Framework_TestSuite->runTest() /usr/share/php/PHPUnit/Framework/TestSuite.php:745
PHP   7. PHPUnit_Framework_TestCase->run() /usr/share/php/PHPUnit/Framework/TestSuite.php:775
PHP   8. PHPUnit_Framework_TestResult->run() /usr/share/php/PHPUnit/Framework/TestCase.php:783
PHP   9. PHP_Invoker->invoke() /usr/share/php/PHPUnit/Framework/TestResult.php:646
PHP  10. call_user_func_array() /usr/share/php/PHP/Invoker.php:93
PHP  11. PHPUnit_Framework_TestCase->runBare() /usr/share/php/PHP/Invoker.php:93
PHP  12. PHPUnit_Framework_TestCase->runTest() /usr/share/php/PHPUnit/Framework/TestCase.php:838
PHP  13. ReflectionMethod->invokeArgs() /usr/share/php/PHPUnit/Framework/TestCase.php:983
PHP  14. AbstractGeneratedCodeTest->testBidiStreaming() /usr/share/php/PHPUnit/Framework/TestCase.php:983
PHPUnit 3.7.28 by Sebastian Bergmann.


Starting test 'GeneratedCodeWithCallbackTest::testSimpleRequest'.
.
Starting test 'GeneratedCodeWithCallbackTest::testServerStreaming'.
.
Starting test 'GeneratedCodeWithCallbackTest::testClientStreaming'.
.
Starting test 'GeneratedCodeWithCallbackTest::testBidiStreaming'.
PHP Fatal error:  Call to a member function getQuotient() on a non-object in /usr/local/google/home/stanleycheung/grpc/src/php/tests/generated_code/AbstractGeneratedCodeTest.php on line 90
PHP Stack trace:
PHP   1. {main}() /usr/local/bin/phpunit:0
PHP   2. PHPUnit_TextUI_Command::main() /usr/local/bin/phpunit:46
PHP   3. PHPUnit_TextUI_Command->run() /usr/share/php/PHPUnit/TextUI/Command.php:129
PHP   4. PHPUnit_TextUI_TestRunner->doRun() /usr/share/php/PHPUnit/TextUI/Command.php:176
PHP   5. PHPUnit_Framework_TestSuite->run() /usr/share/php/PHPUnit/TextUI/TestRunner.php:349
PHP   6. PHPUnit_Framework_TestSuite->runTest() /usr/share/php/PHPUnit/Framework/TestSuite.php:745
PHP   7. PHPUnit_Framework_TestCase->run() /usr/share/php/PHPUnit/Framework/TestSuite.php:775
PHP   8. PHPUnit_Framework_TestResult->run() /usr/share/php/PHPUnit/Framework/TestCase.php:783
PHP   9. PHP_Invoker->invoke() /usr/share/php/PHPUnit/Framework/TestResult.php:646
PHP  10. call_user_func_array() /usr/share/php/PHP/Invoker.php:93
PHP  11. PHPUnit_Framework_TestCase->runBare() /usr/share/php/PHP/Invoker.php:93
PHP  12. PHPUnit_Framework_TestCase->runTest() /usr/share/php/PHPUnit/Framework/TestCase.php:838
PHP  13. ReflectionMethod->invokeArgs() /usr/share/php/PHPUnit/Framework/TestCase.php:983
PHP  14. AbstractGeneratedCodeTest->testBidiStreaming() /usr/share/php/PHPUnit/Framework/TestCase.php:983
stanleycheung@gummybear:~/grpc/src/php$ 
```

**Comment by stanley-cheung** [comment]
Now the bidistreaming test times out

```
stanleycheung@gummybear:~/grpc/src/php$ ./bin/run_gen_code_test.sh 
PHPUnit 3.7.28 by Sebastian Bergmann.


Starting test 'GeneratedCodeTest::testSimpleRequest'.
.
Starting test 'GeneratedCodeTest::testServerStreaming'.
.
Starting test 'GeneratedCodeTest::testClientStreaming'.
.
Starting test 'GeneratedCodeTest::testBidiStreaming'.
E

Time: 3.12 seconds, Memory: 3.50Mb

There was 1 error:

1) GeneratedCodeTest::testBidiStreaming
PHP_Invoker_TimeoutException: Execution aborted after 1 second

/usr/local/google/home/stanleycheung/grpc/src/php/lib/Grpc/BidiStreamingCall.php:58
/usr/local/google/home/stanleycheung/grpc/src/php/tests/generated_code/AbstractGeneratedCodeTest.php:87

FAILURES!
Tests: 4, Assertions: 9, Errors: 1.
```

**Comment by stanley-cheung** [comment]
Looks like this has been fixed by #3144. The PHP math_client test has been run successfully against the ruby math_server
